rbd/rbd.go

```diff
@@ -58,6 +60,8 @@ type ImageInfo struct {
 	Block_name_prefix string
 }
 
+// revive:enable:var-naming
```

Comment:
My revive complains about `Id` below as well. So please move the enabling comment under the `SnapInfo` struct.

Reply:
As discussed in person will be fxed later differently.